<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>543</width>
      <height>640</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_13">
     <item>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Check and tools</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <widget class="QGroupBox" name="groupBox_12">
          <property name="title">
           <string>Find in dictionary</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Codon:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_find_3"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_find_ok_3">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>OK</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_13">
          <property name="title">
           <string>Corresponding list of amino acids</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>List of codons:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_nor_3"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_nor_ok_3">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>OK</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_14">
          <property name="title">
           <string>Add</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_18">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_22">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Codon:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_add_3"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Aminoasid:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_add_4"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>OK</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delete_all_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Delete all codons</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_size_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Size</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_15">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>File</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_19">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_20">
          <item>
           <widget class="QLabel" name="label_24">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Name of file</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item>
             <widget class="QLabel" name="label_25">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Input from:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_file_input_name_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_file_input_ok_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_22">
            <item>
             <widget class="QLabel" name="label_26">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Output to:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_file_output_name_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_file_output_ok_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Ok</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>261</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_21">
     <item>
      <widget class="QListWidget" name="listWidget_3"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete_3">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>440</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
